client/my-sites/plan-features/docs/example.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+
+/**
+ * Internal dependencies
+ */
+import PlanFeaturesHeader from '../header';
+import { plansList, PLAN_PREMIUM } from 'lib/plans/constants';
+
+export default React.createClass( {
+
+	displayName: 'PlanFeatures',
+
+	mixins: [ PureRenderMixin ],
+
+	render() {
+		const price = {
+			currencySymbol: '$',
+			decimalMark: '.',
+			dollars: 8,
+			cents: 25
+		};
+		return (
+			<div className="design-assets__group">
+				<h2>
+					<a href="/devdocs/app-components/plan-features">Plan Features</a>
+				</h2>
+				<div>
+					<PlanFeaturesHeader
+						popular
```

Comment:
I think it would be a good idea if we added one more `<PlanFeaturesHeader>` which would be **not** popular and **not** current. Just to see the diff. :)